src/main/kotlin/com/mdrobnak/lalrpop/resolve/LpPathIdReference.kt

```diff
@@ -0,0 +1,30 @@
+package com.mdrobnak.lalrpop.resolve
+
+import com.intellij.openapi.util.TextRange
+import com.intellij.psi.PsiElement
+import com.intellij.psi.PsiReferenceBase
+import com.intellij.psi.util.PsiTreeUtil
+import com.mdrobnak.lalrpop.psi.LpElementFactory
+import com.mdrobnak.lalrpop.psi.LpNonterminal
+import com.mdrobnak.lalrpop.psi.LpPathId
+
+class LpPathIdReference(element: LpPathId) :
```

Comment:
Why is this added? Path IDs should reference Rust types, not nonterminals.

Reply:
In the case of:
```
NonTerminal<Rule>: Type<Rule> = {}
```

The second `Rule` should references the first

Reply:
It might not be entirely useless, but I'm unsure how it will respond once we inject Rust into the nonterminal type / type refs (#12).

Reply:
So... should I remove it?

Reply:
Sorry, been a bit busy so haven't been able to look at this more until now. We can leave it in until we resolve #12, at which point this may be removed (or it stays helpful for resolving generic types).